apps/storybook/src/Slider.stories.tsx

```diff
@@ -94,7 +94,7 @@ WithCustomThumb.args = {
         alignItems: 'center',
         backgroundColor: '#999',
         width: '36px',
-        height: '26px',
+        height: '20px',
```

Comment:
Just curious, was this necessary?

Reply:
Ohh, I see. But in the second screenshot too, it seems a bit uncentered vertically. Do you know the root cause of the vertical misalignment? Maybe we need to fix that instead?